process-engine/bpmn-studio
# ♻️Refactor BPMN-Studio modules

## What did you change?

This PR refactors the typescript part of the bpmn-studio modules.

## How can others test the changes?

- checkout the branch
- `npm run build`
- `npm run electron-start-dev`
- test the studio with our test form https://goo.gl/forms/83UtLkkIIePkAoWI2
and check whether all works as expected

## PR-Checklist

Please check the boxes in this list after submitting your PR:
